### PR TITLE
[Doc Issue][The provided code to open the scheduling dialog in TeamsJS v2 does not work in Teams][2500275]

### PR DESCRIPTION
**Pull request by v-silakshmi**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/deep-links.md](https://github.com/MicrosoftDocs/msteams-docs/blob/44b219b4c48ad24703295f89180c10686f4c00e9/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdeep-links.md) | 90 | 32 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/18328382-12bc-428c-9e9f-2357f452f40c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/deep-links.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1f77c9b15a837f2862bb7ae84adb07a1c5651c15/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdeep-links.md) | 90 | 32 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ed7c0d63-fcaf-4554-999b-55f03f2eb732) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-habibaumme — DISMISSED** [review]
Approved

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/deep-links.md](https://github.com/MicrosoftDocs/msteams-docs/blob/fb8a4e95fbb8c0ce7cc81bef86d8cc8d0e1e8903/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdeep-links.md) | 90 | 32 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/79f13f7b-b159-4e8e-b6b1-9b419b2f1afc) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/concepts/build-and-test/deep-links.md [hunk 1]
```diff
@@ -204,27 +204,12 @@ microsoftTeams.executeDeepLink(/*deepLink*/);
 
 You can open a scheduling dialog from your Teams app, as shown in the following code. This is especially useful if your app helps the user complete calendar or scheduling related tasks.
```
Comment (ChetanSharma-msft) [inline msteams-platform/concepts/build-and-test/deep-links.md:205]: Can you please remove this line as well?

Comment (v-silakshmi) [reply msteams-platform/concepts/build-and-test/deep-links.md]: Hi! The line is removed.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/deep-links.md](https://github.com/MicrosoftDocs/msteams-docs/blob/44559f64475d25cd74a970dcd334df1d2422baec/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdeep-links.md) | 90 | 30 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e1d73fe1-5a1e-4caa-9b62-1f0f6b13729c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

File: msteams-platform/concepts/build-and-test/deep-links.md [hunk 2]
```diff
@@ -202,29 +202,12 @@ microsoftTeams.executeDeepLink(/*deepLink*/);
...
-      startTime: "2018-10-24T10:00:00-07:00"
-      subject: "test subject"});
-   calendarPromise.
-      then((result) => {/*Successful operation*/}).
-      catch((error) => {/*Unsuccessful operation*/});
-}
-else { /* handle case where capability isn't supported */ }
-```
+> [!NOTE]
+> In order to open the scheduling dialog in Teams, developers need to continue using the original deep-link URL based method, since Teams does not yet support the calendar capability.
```
Comment (ChetanSharma-msft) [inline msteams-platform/concepts/build-and-test/deep-links.md:206]: Can we add the deep link format or reference doc link for better user experience?

**Review by Wajeed-msft — APPROVED** [review]

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**